frontend/libs/studio-components/src/components/StudioResizableLayout/StudioResizableLayoutContainer/StudioResizableLayoutContainer.tsx

```diff
@@ -55,7 +55,11 @@ const StudioResizableLayoutContainer = ({
     >
       <div
         className={classes.root}
-        style={{ ...style, flexDirection: orientation === 'horizontal' ? 'row' : 'column' }}
+        style={{
+          ...style,
+          flexDirection: orientation === 'horizontal' ? 'row' : 'column',
+          maxWidth: '100VW',
+        }}
       >
```

Comment:
Nitpicking here, but we try to avoid inline styling as much as possible. 🙂 I noticed inline styles were used originally as well, but what do you think about refactoring it to something like this?

```
const flexDirectionClass = orientation === 'horizontal' ? classes.horizontal : classes.vertical;
....
....
<div className={`${classes.root} ${flexDirectionClass}`} style={...style}>
....

```
In this case, `maxWidth` could be implemented in classes.root, keeping the layout styling more in the CSS module

Reply:
Thank you, I will try it and make a new commit

Reply:
Well done, @Konrad-Simso 💪